EAD-Session4/Sql/EAD-Session4.sql

```diff
@@ -0,0 +1,17 @@
+drop database parking;
+create database parking;
+use parking;
+
+create table employee(sn int auto_increment primary key, employee_id varchar(20) unique key not null, employee_name varchar(50) not null,
+employee_gender varchar(10) not null, employee_email varchar(100) unique key, employee_password varchar(40), 
+employee_contact varchar(10), employee_oragnization varchar(20), isVehicle int default 0,  isPlan int default 0);
+
+
+create table vehicle (sn int auto_increment primary key, employee_email varchar(100) unique key, 
```

Comment:
We can add a table vehicle, which will contain information about vehicle. As in this case, if vehicle increases, we have to make changes in our code, but if we keep vehicle in db, there won't be any need of making changes in the code.